microsoft/atlas-design
# add initial version of theme switch to Atlas side

Task: task-436851 (buttons)

Link: preview-217

Note: button looks a tad off, because buttons haven't gone into main yet.

## Testing

1. Open preview link.
2. See theme button.
3. Click it to change the theme. Should rotate - light -> dark -> high-contrast -> light
4. Select any non light theme, refresh, it should persist. You should not see a flash.
5. Themes are persisted within local storage only.
Note: if you have prefers dark theme set, you should see it first.
